composer.json

```diff
@@ -14,7 +14,7 @@
         "symfony/framework-bundle" : "~3.0"
     },
     "require-dev" : {
-        "atoum/atoum" : "master-dev",
+        "atoum/atoum" : "dev-master",
```

Comment:
BTW, atoum is not [compatible with PHP 7.1](https://github.com/atoum/atoum/issues/598). So I was not able to run the test suite locally... The issue is almost 1 year old :/